packages/boot/src/types.ts

```diff
@@ -159,3 +159,10 @@ export function booter(artifactNamespace: string, ...specs: BindingSpec[]) {
     ...specs,
   );
 }
+
+/**
+ * Interface to describe an object that nay have an array of `booters`.
```

Comment:
Typo?

```suggestion
 * Interface to describe an object that may have an array of `booters`.
```

Reply:
Thanks for noticing!  I've been typing with some bandages and this has been a frequent occurrence